src_rebuild/EMULATOR/LIBPAD.H

```diff
@@ -35,13 +35,13 @@
 
 #define MAX_CONTROLLERS 2
 
-struct PADRAW
+typedef struct PADRAW
 {
 	unsigned char status; // size=0, offset=0
 	unsigned char id; // size=0, offset=1
 	unsigned char buttons[2]; // size=2, offset=2
 	unsigned char analog[4]; // size=4, offset=4
-};
+} *LPPADRAW;
```

Comment:
I hate seeing this LP* typedefs and it is sometimes quite confusing.
also pls no changes to emulator code as it will be separated for PSX devs

EDIT: that's been put by devs huh